onboarding-site/entrypoint.sh

```diff
@@ -9,7 +9,7 @@ INVENTORY="$(for script in /usr/share/mender/inventory/mender-inventory-*; do $s
 cat >/var/www/localhost/htdocs/device-info.js <<EOF
   mender_server = {
     "Web server": "$(hostname)",
-    "Server address(es)": "[ $(echo "$INVENTORY" | sed -ne '/^ipv4/{s/^[^=]*=//; s,/.*$,,; p}' | tr '\n' ' ')]"
+    "Server address(es)": "[ $(echo "$INVENTORY" | sed -ne '/^ipv4/{s/^[^=]*=//; s,/.*$,,; p}' | tr '\n' ' '):85]"
```

Comment:
This may be a list of addresses, so it won't work correctly, it will look like: `[ 10.0.2.15 192.168.1.15:85 ]`. I think you can just drop this altogether, because this is a purely informational field, and it's not a natural place to list the port number. It's just the server IP address.

Reply:
I agree with @kacf, just remove the port from here